cyhy/mailer/cli.py

```diff
@@ -1060,6 +1153,35 @@ def main():
         )
         return 1
 
+    # Parse out the CSA email mapping, if present.
+    csa_email_file = args["--csa-email-file"]
+    if csa_email_file is not None:
+        try:
+            with open(csa_email_file) as stream:
+                csa_email_mapping = yaml.load(stream, Loader=yaml.SafeLoader)
+        except OSError:
+            logging.critical(
+                f"CSA email file {csa_email_file} does not exist", exc_info=True
+            )
+            return 1
+        except yaml.YAMLError:
+            logging.critical(
+                f"CSA email file {csa_email_file} does not contain valid YAML",
```

Comment:
```suggestion
    csa_emails_file = args["--csa-emails-file"]
    if csa_emails_file is not None:
        try:
            with open(csa_emails_file) as stream:
                csa_email_mapping = yaml.load(stream, Loader=yaml.SafeLoader)
        except OSError:
            logging.critical(
                f"CSA email file {csa_emails_file} does not exist", exc_info=True
            )
            return 1
        except yaml.YAMLError:
            logging.critical(
                f"CSA email file {csa_emails_file} does not contain valid YAML",
```

Reply:
Note @mcdonnnj that I edited your suggestion above to pluralize "emails" in both critical logging messages.